{{cookiecutter.directory_name}}/.github/workflows/build.yml

```diff
@@ -35,8 +35,32 @@ jobs:
           python -m pip install .[dev,publishing]
       - name: Run unit tests
         run: python -m pytest -v
+        run: |
+          python -m coverage run -p -m pytest
       - name: Verify that we can build the package
         run: python -m build
+      - name: Report coverage
+        run: |
+          coverage combine
+          coverage json
+          export TOTAL=$(python -c "import json;print(json.load(open('coverage.json'))['totals']['percent_covered_display'])")
+          echo "total=$TOTAL" >> $GITHUB_ENV
+          echo "### Total coverage: ${TOTAL}%" >> $GITHUB_STEP_SUMMARY
+          echo $'\n```' >> $GITHUB_STEP_SUMMARY
+          coverage report >> $GITHUB_STEP_SUMMARY
+          echo $'\n```' >> $GITHUB_STEP_SUMMARY
+      - name: Make coverage badge
+        if: ${{ github.ref == 'refs/heads/main' && matrix.python-version == '3.12' && matrix.os == 'ubuntu-latest' }}
```

Comment:
I suspect this is what's tripping up Jinja in the failing tests. Try taking the `&&`s out. It probably recognizes this as a Jinja expression and cannot deal with `&`s apparently.

Reply:
Thanks, that makes sense. I didn't immediately recognize these could be jinja2 templates.